src/Microsoft.AspNetCore.Mvc.Razor.Precompilation.Tools/project.json

```diff
@@ -0,0 +1,51 @@
+{
+  "version": "1.0.0-*",
+  "description": "Razor precompilation",
+  "packOptions": {
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/AspNet/Mvc"
+    },
+    "tags": [
+      "cshtml",
+      "razor",
+      "precompilation",
+      "aspnetcore"
+    ]
+  },
+  "buildOptions": {
+    "outputName": "dotnet-razor-precompile",
+    "keyFile": "../../tools/Key.snk",
+    "warningsAsErrors": true,
+    "emitEntryPoint": true,
+    "nowarn": [
+      "CS1591"
+    ],
+    "xmlDoc": true
+  },
+  "dependencies": {
+    "Microsoft.AspNetCore.Hosting": "1.1.0-*",
+    "Microsoft.AspNetCore.Mvc": "1.1.0-*",
```

Comment:
I'm guessing this doesn't depend on the metapackage. It would be nice to be more specific.